web_poet/pages.py

```diff
@@ -7,8 +7,15 @@
 
 
 class Injectable(abc.ABC):
-    """Injectable objects are automatically built and passed as arguments to
-    callbacks that requires them.
+    """
+    Base Page Object class, which all Page Objects should inherit from
+    (probably through Injectable subclasses).
+
+    Frameworks which are using ``web-poet`` Page Objects should use
+    :func:`is_injectable` function to detect if an object is an Injectable,
+    and if an object is injectable, allow building it automatically
+    through dependency injection, using https://github.com/scrapinghub/andi
```

Comment:
We have other references to andi, why now using it here instead of the repository's link?

Reply:
I've put a link here because the docstring may be viewed without Sphinx, and an URL looked easier to understand to me than Sphinx syntax.

Reply:
It makes sense. Thanks.